docs/README.md

```diff
@@ -1,49 +1,52 @@
-# Docs
+# The OPA Website and Documentation
 
-This directory contains all of the Markdown, HTML, CSS, and other assets needed
-to build the [openpolicyagent.org](http://openpolicyagent.org) site. See the
+This directory contains all of the Markdown, HTML, Sass/CSS, and other assets needed
+to build the [openpolicyagent.org](https://openpolicyagent.org) website. See the
 section below for steps to build the site and test documentation changes
 locally.
 
-The raw OPA documentation can be found under the [book](./book) directory.
+The raw OPA documentation can be found under the [content/docs](./content/docs)
+directory.
 
-For development documentation see the [devel](./devel) directory.
+For documentation on developing OPA, see the [devel](./devel) directory.
 
-## Site Updates
-
-We use GitHub pages to host the website that includes all of the OPA
-documentation. In order to update the website, you need to have write permission
-on the open-policy-agent/opa repository.
-
-### Prerequisites
+## Prerequisites
 
 If you want to build and serve the site locally, you need the following packages
 installed on your system:
 
-- npm
-- [gitbook](https://github.com/GitbookIO/gitbook)
+- [npm](https://npmjs.org)
+- The [Hugo](https://gohugo.io/getting-started/installing/) static site generator
 
-### Build and preview the docs locally
+Once npm and Hugo are installed on your system, run `make setup` to install the
```

Comment:
Nit: Would be good to explicitly state that people must install Hugo themselves (and any version requirements)